gestionatr/input/messages/R1.py

```diff
@@ -1091,6 +1119,24 @@ def check(self):
                 errors.append(field)
         return errors
 
+    def check_tipo_disconforme_autoconsumo(self):
+        for var in self.r1.variables_detalle_reclamacion:
+            if not var.disconformidad_autoconsumo:
+                return False
+        return len(self.r1.variables_detalle_reclamacion) > 0
+
+    def check_CAU(self):
+        for var in self.r1.variables_detalle_reclamacion:
+            if not var.CAU:
```

Comment:
```suggestion
            if not var.cau:
```
La variable ha de ser en minúscula, sinó ens surt l'error:
`AttributeError: 'VariableDetalleReclamacion' object has no attribute 'CAU'`